pennylane/templates/subroutines/single_excitation_unitary.py

```diff
@@ -137,9 +137,7 @@ def circuit(weight, ph=None):
 
     ###############
 
-    wires = wires.tolist()  # TODO: Remove when ops accept wires
-
-    r, p = wires
+    r, p = wires.tolist()  # TODO: need to change logic here when introducing non-consec wires
```

Comment:
This still needs to be a list, because the qchem templates heavily depend on wires containing integers which are qubit indices. Need to rewrite those...